### DISCUSSION TITLE
关于《翻译须知》中空格规定的建议

### DISCUSSION THREAD

**Discussion by awxiaoxian2020** [vuejs-translations/docs-zh-cn#585] [Ideas]
在一般情况下，中文输入法即代表全角符号，英文输入法即代表半角符号。

《翻译须知》中规定中文和英文标点有一个空格。而对于括号，没有使用“英文”这一表述，而是使用“半角”的表述。
而这种情况下括号内为中文，括号与中文字符之间应该有一个空格。

实际上是括号内无需空格，而括号外遇到中文得加一个空格。

这种情况是否需要加一个说明。

**Comment by awxiaoxian2020** [comment]
似乎括号不是是否有空格的标志，而是说根据括号内的文字决定？

**Comment by veaba** [comment]
方便举个栗子🌰 吗。

**Comment by veaba** [comment]
大致理解你意思，实际存在中文与英文字符的不恰当配合，导致 markdown 无法正确渲染、操作系统之间差异导致 `全角/半角` 无法正确区别等问题，这引发了一些难以解释一致性的问题，事实上，一方面会依赖 zhlint 来帮忙处理一些格式问题。

**Comment by awxiaoxian2020** [comment]
比如
```text
Vue(读音为...)是...
```
`e` 和括号之间是否需要空格，括号和`读`之间是否需要空格，括号和`是`之间是否需要空格。

因为最近在翻译 Pinia 的文档。

我觉得括号的空格需要单独明确一下
@veaba

**Comment by veaba** [comment]
如果是半角括号，那么 `Vue` 和 `读音` 之间需要空格，全角则不需要。

**Reply by awxiaoxian2020** [reply]
翻译指南已统一为半角括号。

**Comment by Jinjiang** [comment]
我更新了翻译须知 Wiki 的 2.2 - 2.5 部分的描述，补充和完善了一些细节。大家可以参考一下，谢谢
https://github.com/vuejs-translations/docs-zh-cn/wiki/%E7%BF%BB%E8%AF%91%E9%A1%BB%E7%9F%A5
